cluster/charts/crossplane/README.md

```diff
@@ -150,7 +150,7 @@ metadata:
   name: stack-rook
   namespace: rook
 spec:
-  package: "crossplane/stack-rook:master"
+  package: "crossplane/stack-rook:v0.4.0"
```

Comment:
i would expect all the other provider stacks to need to be updated in this file as well.  it looks like even in master, something funny is going on with the build automation to [copy the docs/install-crossplane.md over to this file](https://github.com/crossplaneio/crossplane/blob/master/Makefile#L146) in the helm charts.  we can investigate that in master though.

Reply:
oh i think that `sed` command intends to wipe out the markdown file's front matter, but unintentionally wipes out some example yamls too :(

i'll open an issue for that

Reply:
Gotcha, thanks! We can backport updates :+1: Since this is a straight copy do you want me to handle manually here or no?

Reply:
yes, may as well just copy the file over manually in this PR and update it's refs manually also.

Reply:
@jbw976 I just realized that will cause it to fail `make check-diff` though :disappointed:

Reply:
I have a fix ready for the `sed` command though.

Reply:
ah darn and jenkins calls `make check-diff`.  good point.  well let's just merge this PR as is then without any manual modifications.

Reply:
and we can consider backporting the fix for the `sed` call later on

Reply:
SGTM :heavy_check_mark: